### ISSUE TITLE
Wrong export of code

### ISSUE THREAD

**Issue by jecisc** [pharo-vcs/iceberg#440]
To reproduce this error you need a Pharo 70 image BEFORE the integration of this issue: [https://github.com/pharo-project/pharo/pull/202](https://github.com/pharo-project/pharo/pull/202) (Build 47 of Pharo 7).

This bug was discovered on Windows 7.

I tried to participate to the Pharo development and got a problem. 

Step to reproduce:

Execute this code:

```smalltalk
GTPlayground class compile: 'menuCommandOn: aBuilder
	<worldMenu>
	(aBuilder item: Smalltalk tools workspace title asSymbol)
		parent: #MostUsedTools;
		action: [ Smalltalk tools openWorkspace ];
		order: 0.3;
		keyText: ''o, w'';
		icon: Smalltalk tools workspace taskbarIcon' classified: 'world menu'.
		
Nautilus class compile: 'menuCommandOn: aBuilder
	<worldMenu>
	(aBuilder item: #''System Browser'')
		parent: #MostUsedTools;
		action: [ Smalltalk tools openClassBrowser ];
		order: 0.1;
		keyText: ''o, b'';
		icon: Smalltalk tools browser taskbarIcon' classified: 'world menu'.
		
WorldState class compile: 'mostUsedToolsOn: aBuilder
	<worldMenu>
	(aBuilder group: #MostUsedTools)
		withSeparatorAfter;
		order: 0' classified: 'world menu items'.
		
MCWorkingCopyBrowser class compile: 'menuCommandOn: aBuilder
	<worldMenu>
	(aBuilder item: #''Monticello Browser'')
		parent: #MostUsedTools;
		action: [ Smalltalk tools openMonticelloBrowser ];
		order: 0.9;
		keyText: ''o, p'';
		icon: Smalltalk tools monticelloBrowser taskbarIcon' classified:
'world menu'.
		
TestRunner class compile: 'menuCommandOn: aBuilder
	<worldMenu>
	(aBuilder item: #''Test Runner'')
		parent: #MostUsedTools;
		action: [ Smalltalk tools openTestRunner ];
		order: 0.5;
		keyText: ''o, u'';
		icon: self taskbarIcon' classified: 'world menu'.
		
GTSpotter class compile: 'menuCommandOn: aBuilder
	<worldMenu>
	(aBuilder item: #Spotter)
		action: [ GTSpotterGlobalShortcut openGlobalSpotter ];
		keyText: ''Shift + Enter'';
		order: 0.7;
		parent: #MostUsedTools;
		iconName: #smallFindIcon' classified: 'world menu'.
```

If you try to commit you get this error message:

`Error: Method protocol 'world menu' for the method 'menuCommandOn:' in
class 'GTSpotter' is inconsistent with the package name '*gt-playground'`

If you go in the command line and check the history, the last commit is really messy.

**Comment by syrel** [comment]
Confirmed on OSX too! It is a show stopper. Had to recover repo manually deleting wrong XYZ.extension folders...

**Comment by jecisc** [comment]
In my case it was too messy so I did 6 commits then I squashed them.

**Comment by theseion** [comment]
Should be fixed with PR #452 (see issue #450).
